rust/operator-binary/src/controller.rs

```diff
@@ -378,10 +382,13 @@ mod tests {
     use super::{Context, OpenSearchRoleGroupConfig, ValidatedCluster, ValidatedLogging};
     use crate::{
         controller::{OpenSearchNodeResources, ValidatedOpenSearchConfig},
-        crd::{NodeRoles, v1alpha1},
+        crd::{
+            NodeRoles,
+            v1alpha1::{self, OpenSearchKeystore, SecretKeyRef},
+        },
```

Comment:
We decided to use the versioned module explicitly.


```suggestion
        crd::{NodeRoles, v1alpha1},
```